src/main/scala/com/gu/newsletterlistcleanse/GetCleanseListLambda.scala

```diff
@@ -94,7 +94,7 @@ class GetCleanseListLambda {
 
 object TestGetCleanseList {
   def main(args: Array[String]): Unit = {
-    val json = """{"newsletterName":"Editorial_GuardianTodayUK","cutOffDate":"2020-06-07T11:31:14Z[Europe/London]", "activeListLength": 1000}"""
+    val json = """{"newsletterName":"Editorial_ArtWeekly","cutOffDate":"2020-08-20T11:31:14Z[Europe/London]", "activeListLength": 1000}"""
```

Comment:
I doubt it's a big thing, but does the change here have any particular meaning or is it just left over from manual testing? And what is the expected behaviour now with this particular change?